tox.ini

```diff
@@ -80,11 +80,19 @@ description = Run unit tests
 [testenv:integration]
 commands =
     pip install .
-    pytest {[vars]tst_path}/integration -v --tb native -s {posargs}
+    pytest {[vars]tst_path}/integration/ -m cpu_run -v --tb native -s --notebook-image={env:NOTEBOOK_IMAGE:kubeflownotebookswg/jupyter-scipy:v1.8.0} {posargs}
 deps =
     -r requirements-unit.txt
 description = Run integration tests
 
+[testenv:integration-gpu]
+commands =
+    pip install .
+    pytest {[vars]tst_path}/integration/ -m gpu_run -v --tb native -s --notebook-image=kubeflownotebookswg/jupyter-pytorch-cuda-full:latest {posargs}
+deps =
+    -r requirements-unit.txt
```

Comment:
I missed this before.  Since this requirements is used for both unit and integration tests, should we rename to `requirements-test.txt`?